src/ast/node/pkg.rs

```diff
@@ -361,4 +354,20 @@ impl ExternIdNode {
         }
         Err(ctx.add_diag(self.range.new_err(ErrorCode::MACRO_NOT_FOUND)))
     }
+
+    pub fn solve_mod<'b>(
```

Comment:
document it about how the modules are resolved, seems it tries to retrieve the namespace based on name from the submods inside the `ExternIdNode`